docs/en/operations/settings/settings.md

```diff
@@ -2992,6 +2992,12 @@ Default value: 268435456.
 
 - [min_insert_block_size_bytes](#min-insert-block-size-bytes)
 
+## max_materialized_views_account_for_table {#max_materialized_views_account_for_table}
+
+A limit on the number of materialized views attached to a table.
```

Comment:
iirc, it is possible to create view over another view. let's clarify if we account those outer views or not

Reply:
updated